configure.ac

```diff
@@ -82,6 +86,12 @@ darwin*)
 	LDFLAGS="$LDFLAGS -L/usr/local/lib -L/usr/local/opt/openssl/lib -L/opt/local/lib -L/usr/local/libsrtp/lib"
 	AM_CONDITIONAL([DARWIN_OS], true)
 ;;
+freebsd*)
+	CFLAGS="$CFLAGS -I /usr/include/openssl"
+	LDFLAGS="$LDFLAGS -Xlinker --export-dynamic"
+	LDFLAGS="$LDFLAGS -L/usr/lib/openssl -lcrypto -lssl -L/usr/local/lib"
```

Comment:
could you clarify why crypto and ssl have been removed from PKGCHECKMODULES and manually added to LDFLAGS ?

Reply:
Because FreeBSD comes with openssl in its base system, and without a .pc file.
You can read more here https://lists.freebsd.org/pipermail/freebsd-ports/2020-April/118077.html